CMakeLists.txt

```diff
@@ -50,6 +50,12 @@ else()
   set(GIT_DIR "")
 endif()
 
+find_package(OpenGL)
+if(NOT OPENGL_FOUND)
+  set(CMAKE_VERSION_BUILD "${CMAKE_VERSION_BUILD} (No OpenGL)" CACHE STRING "Build number" FORCE) # git sha
+  message(WARNING "OpenGL libraries were not found! EnergyPlus will be compiled without Pixel Counting functionality for shading calculations!")
```

Comment:
@nealkruis Experience indicates that these exclamation points are going to cause confusion. Could we get rid of those and briefly explain that shading calculations may take longer (or something along those lines)?